sdks/python/apache_beam/ml/inference/base.py

```diff
@@ -1434,19 +1518,28 @@ def load():
     if isinstance(side_input_model_path, str) and side_input_model_path != '':
       model_tag = side_input_model_path
     if self._model_handler.share_model_across_processes():
-      model = multi_process_shared.MultiProcessShared(
-          load, tag=model_tag, always_proxy=True).acquire()
+      # TODO - update this to populate a list of models of configurable length
+      models = []
+      for i in range(self._model_handler.max_shared_model_copies()):
+        models.append(
+            multi_process_shared.MultiProcessShared(
+                load, tag=f'{model_tag}{i}', always_proxy=True).acquire())
+      model_wrapper = _CrossProcessModelWrapper(models, model_tag)
     else:
       model = self._shared_model_handle.acquire(load, tag=model_tag)
+      model_wrapper = _CrossProcessModelWrapper([model], model_tag)
```

Comment:
I thought it was cleaner to just have a single object here so that consumers don't need to worry about it. `_CrossProcessModelWrapper` should be agnostic to which loading method is used.

Maybe this is just a naming issue - does `_SharedModelWrapper` sound cleaner?

Reply:
I tried this, let me know what you think

Reply:
oh, i actually got confused by the `>=0` condition and thought that you intended  to use the multiprocess-shared regardless of # of models. _SharedModelWrapper is better.